lib/qml/app/app.py

```diff
@@ -55,6 +55,7 @@ def build(
         bool, typer.Option(help="Continue if sphinx-build fails for a demo")
     ] = False,
     dev: Annotated[bool, typer.Option(help="Whether to use dev dependencies")] = False,
+    venv: Annotated[Optional[str], typer.Option(help="Name of the virtual environment to install build dependencies")] = None,
```

Comment:
Consistent with Python 3.11+,
```suggestion
    venv: Annotated[str | None, typer.Option(help="Name of the virtual environment to install build dependencies")] = None,
```